packages/web/src/services/audio-player/AudioPlayer.ts

```diff
@@ -141,20 +143,20 @@ export class AudioPlayer {
     this.errorRateLimiter = new Set()
   }
 
-  _initContext = (shouldSkipAudioContext = false) => {
+  _initContext = () => {
     this.audio.addEventListener('canplay', () => {
-      if (!this.audioCtx && !shouldSkipAudioContext) {
+      if (!this.audioCtx) {
         // Set up WebAudio API handles
         const AudioContext = window.AudioContext || window.webkitAudioContext
         try {
           this.audioCtx = new AudioContext()
           this.gainNode = this.audioCtx.createGain()
-          this.gainNode.connect(this.audioCtx.destination)
           this.source = this.audioCtx.createMediaElementSource(this.audio)
           this.source.connect(this.gainNode)
```

Comment:
So moving this after makes a difference?

Reply:
It doesn't actually make a diff, I just thought it was a little more sane to follow what we saw online.

Reply:
hmm really wow